### PR TITLE
Created "edit profile" component. 

### PR DESCRIPTION
**Pull request by yulkats**
1) created component 'edit profile';
2) added route for this page;
3) created subsidiaries components;
4) created HTML markup and placed the components in the correct order;
5) added responsive design;
Expected result(Mock-up, Details at Figma):
1440px
1440px
1024px
1024px
768px
768px
576px
576px
320px
320px

Actual result:
1440px
1440px res (2)
1024px
1024px res (2)
768px
768px res (2)
576px
576px res
320px
320px res (2)

### CODE REVIEW

File: src/app/component/user/components/index.ts [hunk 1]
```diff
@@ -44,3 +48,7 @@ export { AddGoalButtonComponent } from './user-goals/add-goal-button/add-goal-bu
 export { GoalContainerComponent } from './user-goals/goal-container/goal-container.component';
 export { GoalItemComponent } from './user-goals/goal-item/goal-item.component';
 export { GoalListComponent } from './user-goals/goal-list/goal-list.component';
+
+
+
+
```
Comment (Gusvarov) [inline src/app/component/user/components/index.ts:54]: remove these empty lines

File: src/app/component/user/components/profile/edit-profile/personal-photo/personal-photo.component.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    button{
+        border: none;
+        background-color: transparent;
+        left:115px;
+        padding: 0;
+        position: absolute;
+        top: 101px;
+    }
+    
+}
```
Comment (Gusvarov) [inline src/app/component/user/components/profile/edit-profile/personal-photo/personal-photo.component.scss:19]: add newline at the end of a file

File: src/app/component/user/components/profile/edit-profile/social-networks/social-networks.component.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<ul>
+    <li *ngFor="let socialNetwork of socialNetworks">
+        <div class="link">
+            <a href="{{socialNetwork.link}}"><img src="{{socialNetwork.icon}}" alt=""></a>
```
Comment (Gusvarov) [inline src/app/component/user/components/profile/edit-profile/social-networks/social-networks.component.html:4]: Move img tag into a separate line for better code readability

File: src/app/component/user/components/profile/edit-profile/social-networks/social-networks.component.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+                <img src="{{icons.edit}}" alt="edit icon">
+            </button>
+            <button>
+                <img src="{{icons.delete}}" alt="delete icon">
+            </button>
+        </div>
+    </li>
+    <li>
+        <div class="link">
+            <button><img src="{{icons.add}}" alt="add-icon"></button>
```
Comment (Gusvarov) [inline src/app/component/user/components/profile/edit-profile/social-networks/social-networks.component.html:18]: Same as line 4

File: src/app/component/user/components/index.ts (hunk 1 above)
Comment (yulkats) [reply src/app/component/user/components/index.ts]: fixed

File: src/app/component/user/components/profile/edit-profile/personal-photo/personal-photo.component.scss (hunk 2 above)
Comment (yulkats) [reply src/app/component/user/components/profile/edit-profile/personal-photo/personal-photo.component.scss]: fixed

File: src/app/component/user/components/profile/edit-profile/social-networks/social-networks.component.html (hunk 3 above)
Comment (yulkats) [reply src/app/component/user/components/profile/edit-profile/social-networks/social-networks.component.html]: fixed

File: src/app/component/user/components/profile/edit-profile/social-networks/social-networks.component.html (hunk 4 above)
Comment (yulkats) [reply src/app/component/user/components/profile/edit-profile/social-networks/social-networks.component.html]: fixed

File: src/app/component/user/components/profile/edit-profile/edit-profile.component.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
+<div class="container">
+    <div class="title">
+        <h2>Edit profile</h2>
+    </div>
+    <app-personal-photo></app-personal-photo>
+    <form>
+        <div class="wrapper">
+            <p>Personal info</p>
+            <div class="input-block" [ngStyle]="{'margin-bottom': '41px'}">
```
Comment (RomanDeveloperAcc) [inline src/app/component/user/components/profile/edit-profile/edit-profile.component.html:9]: General: please, if you have more than 1 attribute, write them in a separate rows, for better code readability.

File: src/app/component/user/components/profile/edit-profile/edit-profile.component.scss [hunk 6]
```diff
@@ -0,0 +1,177 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');
+
+.container {
+    margin-left: 135px;
+    max-width: 554px;
+    box-sizing: content-box;
+
+    h2 {
+        font-family: Montserrat, sans-serif;
```
Comment (RomanDeveloperAcc) [inline src/app/component/user/components/profile/edit-profile/edit-profile.component.scss:10]: It is better to use scss variables for fonts

File: src/app/component/user/components/profile/edit-profile/edit-profile.component.html (hunk 5 above)
Comment (yulkats) [reply src/app/component/user/components/profile/edit-profile/edit-profile.component.html]: fixed

File: src/app/component/user/components/profile/edit-profile/edit-profile.component.scss (hunk 6 above)
Comment (yulkats) [reply src/app/component/user/components/profile/edit-profile/edit-profile.component.scss]: fixed

**Pull request merged**